src/plugins/auto/plugin.cpp

```diff
@@ -106,8 +106,12 @@ std::vector<DeviceInformation> MultiDeviceInferencePlugin::ParseMetaDevices(cons
         if (!deviceIDLocal.empty()) {
             tconfig[PluginConfigParams::KEY_DEVICE_ID] = deviceIDLocal;
         }
-
-        return GetCore()->GetSupportedConfig(deviceName, tconfig);
+        auto deviceConfig = GetCore()->GetSupportedConfig(deviceName, tconfig);
+        if (deviceConfig.find(PluginConfigParams::KEY_PERFORMANCE_HINT) == deviceConfig.end() && tconfig.find(deviceName) == tconfig.end()) {
+            // setting tput as the default performance mode if no hints setting for AUTO plugin and no properties specified for target device.
+            deviceConfig[PluginConfigParams::KEY_PERFORMANCE_HINT] = PluginConfigParams::THROUGHPUT;
```

Comment:
I found that benchmark_app -d MULTI:GPU -hint latency  will trigger tput as well, is this as expected?
@yangwang201911

Reply:
@peterchen-intel @yangwang201911 can we test on DG1 to see if this functions well?

Reply:
Below steps could reproduce this issue. The root cause for this exception is ` deviceConfig = GetCore()->GetSupportedConfig(deviceName, tconfig); ` didn't retrieve the config setting for CPU plugin. Function `GetSupportedConfig(deviceName, tconfig);` just merge the config passed through `compiled_model` (including the secondary level properties) if this config is one of the supported keys by CPU plugin. We need to update the function `GetSupportedConfig();` to merge the configs set via function `set_property()` before calling `compiled_model()` into the final configs list for "CPU" plugin. Is this ok to fix this issue? @peterchen-intel @wangleis 

`
core.get_property("CPU", ov::supported_properties);
core.get_property("MULTI", ov::supported_properties);
core.set_property("CPU", {ov::hint::performance_mode(ov::hint::PerformanceMode::LATENCY)});
ov::CompiledModel compiled_model = core.compile_model(model, "MULTI:CPU");
`